sources/api/netdog/src/cli/install.rs

```diff
@@ -50,36 +50,60 @@ pub(crate) fn run(args: InstallArgs) -> Result<()> {
     }
 
     match (&args.interface_type, &args.interface_family) {
-        (InterfaceType::Dhcp, InterfaceFamily::Ipv4 | InterfaceFamily::Ipv6) => {
+        (interface_type @ InterfaceType::Dhcp, InterfaceFamily::Ipv4 | InterfaceFamily::Ipv6) => {
```

Comment:
In the "static" match arm we pull the static lease instead of the DHCP lease, otherwise the actions are the same.  

I suppose I could move the `InterfaceFamily` enum somewhere and use it more globally, and then use that enum as an argument to a "get lease" function

Reply:
Yes, you're right - I'll do that!